docs/docs-content/clusters/public-cloud/aws/eks.md

```diff
@@ -1,35 +1,40 @@
 ---
 sidebar_label: "Create and Manage AWS EKS Cluster"
 title: "Create and Manage AWS EKS Cluster"
-description: "Learn how to deploy and manage AWS EKS clusters with Palette"
+description: "Learn how to deploy and manage AWS EKS clusters with Palette."
 hide_table_of_contents: false
-tags: ["public cloud", "aws"]
+tags: ["public cloud", "aws", "eks"]
 sidebar_position: 30
 ---
 
 
-Palette supports creating and managing AWS Elastic Kubernetes Service (EKS) clusters deployed to an AWS account. This section guides you on how to create an AWS EKS cluster in AWS that is managed by Palette.
+Palette supports creating and managing Amazon Web Services (AWS) Elastic Kubernetes Service (EKS) clusters deployed to an AWS account. This section guides you on how to create an EKS cluster in AWS that Palette manages.
 
 ## Prerequisites
 
-The following prerequisites must be met before deploying a cluster to AWS:
+- Access to an AWS cloud account.
 
-- Access to an AWS cloud account 
 - Palette integration with AWS account. Review [Add AWS Account](add-aws-accounts.md) for guidance.
-- An infrastructure cluster profile for AWS EKS. Review [Create an Infrastructure Profile](../../../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md) for guidance.
-- An [EC2 Key Pair](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) in the target region.
-- Palette creates compute, network, and storage resources in AWS during the provisioning of Kubernetes clusters. Ensure there is sufficient capacity in the preferred AWS region for the creation of the following resources:
-    - vCPU
-    - VPC
+
+- An infrastructure cluster profile for AWS EKS. When you create the profile, ensure you choose **EKS** as the **Managed Kubernetes** cloud type. Review [Create an Infrastructure Profile](../../../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md) for guidance.
+
+- An EC2 key pair for the target region that provides a secure connection to your EC2 instances. To learn how to create a key pair, refer to the [Amazon EC2 key pairs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) resource.
+
+- kubelogin installed. This is a kubectl plugin for Kubernetes OpenID Connect (OIDC) authentication, also known as `kubectl oidc-login`.
+
+- Palette creates compute, network, and storage resources in AWS when it provisions Kubernetes clusters. Ensure there is sufficient capacity in the preferred AWS region to create the following resources:

```

Comment:
This is true, but the user has the ability to specify an existing VPC. If they provide their own VPC, then Palette won't create the resources below.

Reply:
added clarification.